1.10.0 Release

endervad opened the pull request:
* DT: До Ур. 100
* Хильдибранд (Heavensward)
* Названия сокровищниц

TinySkald committed:
Текстуры для MSQ 7.0

- Все оставшиеся текстуры для зон, подземелий и испытаний 95-100 уровней

TinySkald committed:
текстурки карт сокровищ

TinySkald committed:
7.0 пост титры

TinySkald committed:
хильдибранд ХС

endervad committed:
Update submodule URL

TinySkald committed:
Текстуры 18.05.25

- Данжи, триал 7.1-7.2
- Текстуры окна помощи (по запросу Коннора)
- Пропущенные текстуры Райского Острова
- Большая часть текстур Космических исследований (все еще остались 3-4 текстуры в духе "Код Красный)

TinySkald committed:
текстурки 20.05

- одна текстурка для названия локации космического исследования

TinySkald committed:
Текстуры 01.06

Добавлены текстуры из канала #текстуры-ошибки

TinySkald committed:
Текстурки 25.06

Замена ряда текстур данжей АРР в связи с их переименовкой

TinySkald committed:
17.08 текстуры

- Адский Кьер - изменен на Омут Ада
- Часть текстур для Оккультного Полумесяца

TinySkald committed:
7.3 обновка

текстуры к данжу и триалу + послетитровые

TinySkald committed:
Апдейт 21.10

- Файл фш с боссами
- ПВП победа и поражение (у них нету не HD) версии
-Замена имен данжей и триалов в соответствии с новыми изменениями имен
-самая важная текстура в игре: блю маг
-хильдибранд СБ
- Боссы (всё МСК ShB, парочка текстур боссов из ЭВ и ДТ)

TinySkald committed:
23.10

- Комбо текстуры Оккультного Полумесяца
- Текстура волшебного горшка оттуда же
- ФШ файл для будущей текстуры Призрачных Оружий

TinySkald committed:
Текстурки 17.12

- Текстуры Глубоких Подземелий
- Фиксы ошибок (Гора Гулг, Райский остров)
- Текстуры свадьбы
- Текстуры репутаций
- Текстура релика ДТ
- Текстуры туринров Тройного Трио

TinySkald committed:
09.01.2026

- текстуры золотой араи для сюжетки ЭВ хильдибрана

TinySkald committed:
Обнова 23.01

-Текстуры данжа и триала 7.4
-Ещё текстуры ДТ релика

TinySkald committed:
28.01

- Данжи EW (до 6.0)

TinySkald committed:
07.02

праздничные текстуры

adastmin1 commented:
[exd_conversion](https://github.com/xivrus/exd_conversion)     

 : exh
1 EXHs found.
Do you want to make choices for all file? (Y/n): Y
Do you want to overwrite existing CSVs?
ПРЕДУПРЕЖДЕНИЕ: You will lose current translations in overwritten files!
(y/N) : y
Do you want to copy source to translation?
Choosing 'n' will leave 'Translation' field empty (Y/n): y
Do you want to add '<index>_' at the start of 'Translation' fields? (y/N): y
Write-Warning : Не удается найти параметр, соответствующий имени параметра "ForegroundColor".
G:\C9\exd_conversion-main\_EXDtoCSV.ps1:280 знак:21
+                     -ForegroundColor DarkRed *>&1 | Tee-Object $log - ...
+                     ~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidArgument: (:) [Write-Warning], ParameterBindingException
    + FullyQualifiedErrorId : NamedParameterNotFound,Microsoft.PowerShell.Commands.WriteWarningCommand

Finished in 00:00:00.119

Encountered the following errors:
item

TinySkald committed:
30.03

-две несчастные текстурки

TinySkald committed:
30.04

- текстуры для космоса

TinySkald committed:
Убираю ошибочные файлы

TinySkald committed:
11.06.2026

-Особые доставки
-Союзные задание
-Текстуры звезд для космоса
-Текстуры открытия морей для ФК корабля
-Различные фиксы

TinySkald committed:
12.06

-FFXI ивентовые текстуры
-текстуры горшков Золотого Блюдца
-Текстуры ДТ доставок
